Ix.NET/Source/System.Linq.Async/System.Linq.Async.csproj

```
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Condition="'$(TargetFramework)' != 'netcoreapp3.1' and '$(TargetFramework)' != 'netstandard2.1' " Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
+    <PackageReference Condition="'$(TargetFramework)' == 'net48' or '$(TargetFramework)' == 'netstandard2.0' " Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

If we only want to include this package reference on non-.NET Core targets, the idiomatic condition would be:

```suggestion
    <PackageReference Condition=" '$(TargetFrameworkIdentifier)' != '.NETCoreApp' " Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

Note that net5.0, net6.0, etc fall under `.NETCoreApp` too.

See some examples of this condition in [the dotnet/runtime repo](https://github.com/search?q=repo%3Adotnet%2Fruntime%20TargetFrameworkIdentifier%20path%3A*.csproj&type=code).